tests/safety/common.py

```diff
@@ -18,16 +17,12 @@ class UNSAFE_MODE:
 
 def package_can_msg(msg):
   addr, _, dat, bus = msg
-  rdlr, rdhr = struct.unpack('II', dat.ljust(8, b'\x00'))
-
-  ret = libpandasafety_py.ffi.new('CAN_FIFOMailBox_TypeDef *')
-  if addr >= 0x800:
-    ret[0].RIR = (addr << 3) | 5
-  else:
-    ret[0].RIR = (addr << 21) | 1
-  ret[0].RDTR = len(dat) | ((bus & 0xF) << 4)
-  ret[0].RDHR = rdhr
-  ret[0].RDLR = rdlr
+  ret = libpandasafety_py.ffi.new('CANPacket_t *')
+  ret[0].extended = 1 if addr >= 0x800 else 0
+  ret[0].addr = addr
+  ret[0].len = len(dat)
+  ret[0].bus = 3 if bus > 3 else bus
```

Comment:
@robbederks and @adeebshihadeh , any ideas why/how I am getting weird bus numbers on input to package_can_msg function? It is 832 on Jenkins machine and 579 on my home workstation every run. Of course I am getting overflow as bus now limited to 2 bits. With the same func everything is fine on safety replay.